### PR TITLE
Orbidder Bid Adapter: add native support

### PR DESCRIPTION
**Pull request by arneschulz1984**
## Type of change
- [x] Feature

## Description of change
Adding native support to orbidder adapter

### Contact email 
- REALTIME-SIGGI@otto.de

### CODE REVIEW

File: modules/orbidderBidAdapter.md [hunk 1]
```diff
@@ -12,19 +12,46 @@ Module that connects to orbidder demand sources
...
+var adUnits = [
+    {
+        code: 'test_banner',
+        mediaTypes: {
+            banner: {
+                sizes: [728, 90]
+            }
+        },
+        bids: [{
+            bidder: 'orbidder'
```
Comment (aleksatr) [inline modules/orbidderBidAdapter.md:24]: Hi @arneschulz1984, missing `,` here, on line 49 and line 51, everything else looks good to me.

Comment (arneschulz1984) [reply modules/orbidderBidAdapter.md]: Hi @aleksatr, thanks for the hint. Fixed to valid JavaScript with commit: https://github.com/prebid/Prebid.js/pull/7047/commits/e6e824403b3f150d85939ebed943a8c8abf8cb2e

**Comment by arneschulz1984** [comment]
Hi, one short question, do we have to do something here?
The Circle CI job failed, but we only changed some documentation.
Should i trigger ab build again by pushing a new commit?
Greetings

**Comment by aleksatr** [comment]
> Hi, one short question, do we have to do something here?
> The Circle CI job failed, but we only changed some documentation.
> Should i trigger ab build again by pushing a new commit?
> Greetings

Since the error was not linked to your adapter, I just reran the CircleCI job and now you have your build/tests passing.

**Comment by arneschulz1984** [comment]
Hi,
can you give me a notice, with witch release our feature will be released.
Our partners are waiting for our native supporting adapter.

Greetings

**Comment by aleksatr** [comment]
hi @arneschulz1984 , sorry for the delay, it will be released in 5.4.0.

**Comment by arneschulz1984** [comment]
> hi @arneschulz1984 , sorry for the delay, it will be released in 5.4.0.

Hi @aleksatr, thanks a lot!
Have a nice day

**Pull request merged**